doc/openapi-user-guide.md

```diff
@@ -194,7 +194,14 @@ spec:
   * Only first `servers[0].url` element in `servers` list parsed as *private base url*. As OpenAPI specification `basePath` property, `servers[0].url` URL's base path component will be used.
   * `servers` element in path item or operation items are not supported.
   * Just a single top level security requirement supported. Operation level security requirements not supported.
-  * Supported security schemes: `apiKey`.
+  * Supported security schemes: `apiKey`. 
+* As detailed in section [OpenID Connect and OAuth2 example](#openid-connect-and-oauth2-example)  - 
+3scale requires some additional information that are not included in the OpenAPI spec, so need to be provided in the OpenAPI CR:
+  * OpenID Connect Issuer Type, which will default to rest but can be overridden from the OpenAPI CR 
+  * OpenID Connect Issuer Endpoint Referense (Secret). 3scale requires that the issuer URL must include a client secret. 
+  * Flows object. When the sec scheme is oauth2, the flows are provided by the OpenAPI doc. However, for openIdConnect security scheme, the OpenAPI doc does not provide the flows. In that case, the OpenAPI CR can provide those.
```

Comment:
is `sec` security? 
if OpenAPI does not provide the flows, does OpenAPI must provide them or is it optional? If it must, change "can" to "must" please. If not it's fine

Reply:
It's optional. The default is false. As noted in Design: https://github.com/3scale/3scale-operator/issues/842 , where also example for OpenAPI CRD.  In code - it's here: https://github.com/3scale/3scale-operator/blob/master/controllers/capabilities/openapi_product_reconciler.go#L399-L403
Thank you